edf/FindForm.cs

```diff
@@ -4,15 +4,24 @@
 using System.IO;
 using System.Reflection;
 using System.Windows.Forms;
+using System.Text.Json;
 
 
 namespace SubReal.EasyDuplicatesFinder
 {
     public partial class FindForm : Form
     {
-        private EdfFiles _edfFiles;
+        private EdfFiles _edfFiles;     
+        
+        const string optionsFileName = "optionsefd.json";
```

Comment:
Typo in file name? I think we could use 'edf-settings.json' instead.

Reply:
yes is it. I agree with the suggested name.